tonic_validate/metrics/contains_text_metric.py

```diff
@@ -18,7 +18,8 @@ def __init__(
         case_sensitive: Optional[bool] = False,
     ):
         """
-        Create a metric that checks if the LLM response contains a given string.
+        Create a binary metric that checks if the LLM response contains a given string.
+        Returns 1 (True) if the LLM response contains the given string, 0 (False) otherwise.
```

Comment:
Returns 1 (True) if the LLM response contains the string. Returns 0 (False) if it does not contain the string.